src/board/system76/galp5/board.mk

```diff
@@ -36,23 +36,19 @@ CFLAGS+=\
 	-DPOWER_LIMIT_DC=28
 
 # Custom fan curve
-CFLAGS+=-DBOARD_HEATUP=5
-CFLAGS+=-DBOARD_COOLDOWN=20
 CFLAGS+=-DBOARD_FAN_POINTS="\
-	FAN_POINT(70, 40), \
-	FAN_POINT(75, 60), \
-	FAN_POINT(80, 75), \
-	FAN_POINT(85, 90), \
-	FAN_POINT(90, 100) \
+	FAN_POINT(70, 25), \
```

Comment:
This is the only fan curve I updated. these are the values I run and they correspond to the system76-power profiles for battery, balanced and performance.

Reply:
You're defining multiple speeds for some of the temperatures here. It seems like the speed is changing around 80, so I take it the highest speed defined for a temperature (or the last point defined for a temperature) takes precedence?

This curve is much quieter, but that's probably more to do with the speeds being nearly cut in half than the smoothing. Running a simple stress test (`stress -c 4 -i 4 -m 4 -d 4`), I'm getting temperatures hovering just above 80 degrees with the new curve, while the old curve hovered a few degrees lower in the high 70's. With the new curve, switching to battery mode brings the temperature below 70 almost immediately, with the fans spinning down soon after. Performance mode sits in the low 90's, keeping the fans at 100%.

Overall, I think this curve will result in a quieter experience for most users. Some users might have a concern about the slightly increased temperatures, but I'd expect the complaints about that to be less than the current complaints about noise levels. galp5's plastic bottom panel won't transmit the heat to the user's lap as much as the metal panel on the lemp10.

Even with this curve, I'm still able to create the behavior (especially in performance mode) where I start a stress test, wait for the fans to start spinning up, stop the stress test, then hear the fans continue to spin up well after the temperature's dropped below 60. I counted six seconds between when the temperature began to drop and when the fans began to drop. With `BOARD_COOLDOWN` and `BOARD_DGPU_COOLDOWN` lowered from 20 to 10, I'm not able to recreate that so easily, as the fans drop down to 40% very soon after the temperatures start falling.

I also see that this PR was what increased the common values for `BOARD_COOLDOWN` and `BOARD_DGPU_COOLDOWN` to 20 in the first place, although I assume it was done because most of the models were overriding with a cooldown of 20. In any case, I would still recommend bringing those down to 10, at least for galp5 where the curve points are so far apart from each other in temperature (making bouncing between points less likely.)

Happy to have @leviport or @bflanagin take a second look on our lab unit, this still feels fairly subjective even when I'm watching the numbers.

Reply:
`10` is also the ideal value for the Oryp7 (and by extension, the Oryp6), as I mentioned a few weeks back.  I know I don't work at System76, but here's one random Internet person's support for making that value default.  :)

Reply:
@jacobgkau thanks for the detailed report! I wouldn't be upset if S76 modified default values of course. The effective cooldown time we currently default to is about 5 seconds nominally (6 as you observed though) which is still down considerably from the 10 that we previously defaulted to. It's a good point that the smoothing will take another 10 seconds or so bring the fans all the way to zero, but I was more concerned with bounciness of the fans than the race to zero after load. Hopefully these values will smooth short spikes in load entirely and reduce bounciness for constant loads. There's infinite variability in workloads of course, but I've singled out what I believe are the two most common scenarios and announces associated with them:
1. short spike in load should not result in short spike in fan speeds
2. constant elevated load should not result in fan speeds bouncing noticeably

Reply:
@jacobgkau and in response to the multiple fan points, I think you understand correctly the intention to create a pivot point in the fan curve. In theory, we can set just two fan points, a fan speed floor and ceiling and all values in between would interpolate, but to better match with the S76 power profiles, I've added these values

Reply:
I also have to agree that I like a cooldown time of `10` on my oryp6. `20` just feels like it goes way too long after a load spike, and the CPU temps have long since dropped off by the time the fan starts smoothing downwards.

@curiousercreative If you want to push that commit, I think this PR is just about ready to be approved.

Reply:
I went ahead and committed that change (we're excited to get these improvements in!), I'll make sure that took effect and then we'll approve this.